src/bridge/index.ts

```
@@ -66,6 +66,15 @@ const isOsShareMessage = (
   return type === 'os:share';
 };
 
+/**
+ * 메시지 타입이 OsShare 관련인지 확인 (Type Guard)
+ */
+const isOsShareMessage = (
```

Biome (2.4.4)

[error] Shouldn't redeclare 'isOsShareMessage'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)